transaction-service/transaction-module/src/main/kotlin/com/ajaxsystems/application/useCases/FindTransactionsRestApiInputPort.kt

```diff
@@ -0,0 +1,17 @@
+package com.ajaxsystems.application.useCases
+
+import com.ajaxsystems.domain.dto.TransactionDto
+import java.time.LocalDateTime
+import java.util.UUID
+import org.springframework.data.domain.Pageable
+import reactor.core.publisher.Mono
+
+interface FindTransactionsRestApiInputPort {
```

Comment:
RestApi in interface name bounds it to infrastructure, but this is application layer which should know nothing about infrastructure

Reply:
agree